README.md

```diff
@@ -1,28 +1,33 @@
 ## C# for Visual Studio Code
-A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marketplace.visualstudio.com/VSCode) that provides rich language support for C# and is shipped as part of [C# Dev Kit][csdevkitextension]. 
-
-This pre-release version of the extension is available for use today, and offers performant and reliable support for coding in C#. Powered by a Language Server Protocol (LSP) Tools Host, this extension integrates with open source components like [Roslyn](https://github.com/dotnet/roslyn) and [Razor](https://github.com/dotnet/razor) to provide rich type information and a faster, more reliable C# experience.
+A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marketplace.visualstudio.com/VSCode) that provides rich language support for C# and is shipped as part of [C# Dev Kit][csdevkitextension]. Powered by a Language Server Protocol (LSP) server, this extension integrates with open source components like [Roslyn](https://github.com/dotnet/roslyn) and [Razor](https://github.com/dotnet/razor) to provide rich type information and a faster, more reliable C# experience.
 
 ## Recommended Install
 While it is possible to use the C# extension as a standalone extension, we highly recommend using [C# Dev Kit][csdevkitextension].
 
-1. Installing [C# Dev Kit][csdevkitextension] will automatically install this extension as a required dependency
+1. Installing [C# Dev Kit][csdevkitextension] will automatically install this extension as a required dependency.
 2. Open a folder/workspace that contains a C# project (.csproj) and a C# solution (.sln) and the extension will activate.
 3. Whether you install C# Dev Kit or just the C# extension, the [.NET Runtime Installer Tool extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime) will be installed as a dependency.
 
-Note: If working on a solution that requires versions prior to .NET 6, install a Full Framework runtime and MSBuild tooling.
-  * Set omnisharp.useModernNet to false and if you are on the pre-release verion, set dotnet.server.useOmnisharp to true
-  * Windows: .NET Framework along with MSBuild Tools 
-  * MacOS/Linux: Mono with MSBuild 
+Note: If working on a solution that requires versions prior to .NET 6 or if you have a Unity project, install a Full Framework runtime and [MSBuild tooling](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
```

Comment:
are we planning to update the unity portion of the recommendation later in the week?